dataprofiler/tests/profilers/test_int_column_profile.py

```diff
@@ -366,6 +370,7 @@ def test_profile_merge(self):
         expected_profile = dict(
             min=2.0,
             max=15.0,
+            sum=33,
```

Comment:
Once again  not sure how 33 is the sum here, not saying it isn't correct, I may just be ignorant to the calculation details

Reply:
As above, the float doesn't count.